cmd/utils/flags.go

```diff
@@ -1490,6 +1495,8 @@ func SetKlayConfig(ctx *cli.Context, stack *node.Node, cfg *cn.Config) {
 	cfg.RestartTimeOutFlag = ctx.GlobalDuration(RestartTimeOutFlag.Name)
 	cfg.DaemonPathFlag = ctx.GlobalString(DaemonPathFlag.Name)
 
+	cfg.RPCGasCap = new(big.Int).SetUint64(ctx.GlobalUint64(RPCGlobalGasCap.Name))
```

Comment:
We need to check the flag is set or not.
Because, default is now gas cap. In that case, `cfg.RPCGasCap` should be nil.

Reply:
As it is checked by `hi > gasCap.Uint64()`, I think 0 value can be used without checking. 
Please correct me if I am wrong 😅

Reply:
I agree with @KimKyungup. When the flag is not set, `cfg.RPCGasCap` and `hi` will be set to `0` also.

Reply:
Using default `0` value is not right for `doCall` and `esitmateGas`. 
In geth PR, there is a routine to check. We shouldn't skip it.

Reply:
Understood. Updated!